schema.json

```diff
@@ -0,0 +1,264 @@
+{
+    "workshop": {
+	"type": "dict",
+	"required": true,
+	"schema": {
+	    "schema": {
```

Comment:
Is this a field name or some meta-data for cerberus?  It's hard to tell with schema->schema

Reply:
The inner property is actually a field name so that the resulting JSON that is accepted has schema as an allowed property:

```
{
    "workshop": {
        "schema": {
            "version": "2020.03.02"
        }
    },
...
```